tests/common/reboot.py

```diff
@@ -223,3 +230,48 @@ def check_reboot_cause(dut, reboot_cause_expected):
     reboot_cause_got = get_reboot_cause(dut)
     logging.debug("dut {} last reboot-cause {}".format(dut.hostname, reboot_cause_got))
     return reboot_cause_got == reboot_cause_expected
+
+
+def check_reboot_cause_history(dut, reboot_type_history_queue):
+    """
+    @summary: Check the reboot cause history on DUT. Can be used with wailt_until
+    @param dut: The AnsibleHost object of DUT.
+    @param reboot_type_history_queue: reboot type queue.
+    e.g.
+    show reboot-cause  history
+    Name                 Cause          Time                             User    Comment
+    -------------------  -------------  -------------------------------  ------  ---------
+    2021_09_09_14_15_13  Power Loss ()  N/A                              N/A     N/A
+    2021_09_09_14_06_17  reboot         Thu 09 Sep 2021 02:05:17 PM UTC  admin   N/A
+    2021_09_09_13_59_11  Watchdog ()    N/A                              N/A     N/A
+    2021_09_09_13_52_13  Power Loss ()  N/A                              N/A     N/A
+    2021_09_09_13_45_18  warm-reboot    Thu 09 Sep 2021 01:44:14 PM UTC  admin   N/A
+    2021_09_09_13_37_58  fast-reboot    Thu 09 Sep 2021 01:37:09 PM UTC  admin   N/A
+    2021_09_09_13_30_52  soft-reboot    Thu 09 Sep 2021 01:30:24 PM UTC  admin   N/A
+    2021_09_09_13_24_17  reboot         Thu 09 Sep 2021 01:23:17 PM UTC  admin   N/A
+    """
+    reboot_cause_history_got = dut.show_and_parse("show reboot-cause history")
```

Comment:
show reboot-cause history is a new feature in 202012, can we make it a new test and skip the test for release before 202012?

Reply:
Because reboot consume too much time, so I don't  create a new test for it, and want to leverage the reboot-cause history generated by them to test it.
can we add an version check when version is before 202012, skip the test. Is it ok?

Reply:
ok to skip the check

Reply:
Done